Dockerfile

```diff
@@ -30,12 +30,14 @@ COPY --parents .yarn/ yarn.lock .yarnrc.yml **/package.json packages/bind-mount/
 #
 # If the following issue is ever addressed, we can use that instead:
 # https://github.com/yarnpkg/berry/issues/6339
-RUN cd /PrairieLearn && yarn dlx node-gyp install && yarn install --immutable --inline-builds && yarn cache clean
+WORKDIR /PrairieLearn
+RUN yarn dlx node-gyp install && yarn install --immutable --inline-builds && yarn cache clean
 
 # NOTE: Modify .dockerignore to allowlist files/directories to copy.
-COPY . /PrairieLearn/
+COPY . .
 
 # set up PrairieLearn and run migrations to initialize the DB
+# hadolint ignore=DL3003,SC3009
```

Comment:
You can fix the `DL3003` violation by removing `cd /PrairieLearn` from below, which I believe is now unnecessary since we have `WORKDIR /PrairieLearn` above.

Reply:
Fixed in 5ae834225dd4c7b110bee80a810331ddd36ee375.